.travis.yml

```diff
@@ -6,5 +6,7 @@ rvm:
   - 2.2
   - 2.3
```

Comment:
Also, I think, you can drop 2.3 and earlier, because the maintenance (even security) has been ended: https://www.ruby-lang.org/en/news/2019/03/31/support-of-ruby-2-3-has-ended/